### ISSUE TITLE
Is there an introduction of requirement?

### ISSUE THREAD

**Issue by wangyuxin87** [jichilen/SAR-pytorch#2]
Hi,
  Thanks for your contribution. Is there an introduction of requirement? It may help me run your code.

**Comment by wangyuxin87** [comment]
Such as pytorch 1.0 and so on.

**Comment by jichilen** [comment]
Ok, first i want to show you some useful informations, this branch is not exactly the same with the official torch implementations. Such as padding and masksoftmax and maybe attention machanism. What i can guarantee is that we do use it in ICDAR19 compatitions and it can achieve results which are not so bad. You can refer to https://github.com/Pay20Y/SAR_TF, which is a more useful branch which is implemented by tensorflow. (most of the details are reimplemented following the official torch version)
As for installation requirements:
Python 3.5
opencv-python
torch==1.1.0
torchvision==0.2.1
tqdm
editdistance==0.5.2
torchsummary(not used but imported)
numpy 
scipy(only for postfix .mat)

**Comment by wangyuxin87** [comment]
> (most of the details are reimplemented following the official torch versi

Thanks. I think these will help me a lot.
